code/modules/admin/admin_verbs.dm

```diff
@@ -322,10 +322,9 @@ var/list/admin_verbs_hideable = list(
 	if(istype(mob,/mob/dead/observer))
 		//re-enter
 		var/mob/dead/observer/ghost = mob
-		if (!ghost.can_reenter_corpse)
+		if (ghost.can_reenter_corpse)
 			log_admin("[key_name(usr)] re-entered corpse")
 			message_admins("[key_name_admin(usr)] re-entered corpse")
-		ghost.can_reenter_corpse = 1			//just in-case.
```

Comment:
Also, this breaks the intended functionality.

I made aghost announcing this way so that aghost would only announce re-entering your body if you couldn't normally re-enter your body,